### PR TITLE
Update test_move_feature

### PR DESCRIPTION
**Pull request by jgersak**
parametrize
add types
use existing fixture

### CODE REVIEW

File: core/eolearn/tests/test_core_tasks.py [hunk 1]
```diff
@@ -259,52 +260,25 @@ def test_initialize_feature(patch):
...
+@pytest.mark.parametrize(
+    "features, deep",
+    [
+        ([(FeatureType.DATA, "bands")], False),
+        ([(FeatureType.DATA, "bands"), (FeatureType.MASK_TIMELESS, "mask")], True),
+        ([(FeatureType.DATA, "bands"), (FeatureType.BBOX, None)], False),
+    ],
+)
+def test_move_feature(features: FeatureSpec, deep: bool, patch: EOPatch) -> None:
+    patch_dst = MoveFeatureTask(features, deep_copy=deep)(patch, EOPatch())
```
Comment (zigaLuksic) [inline core/eolearn/tests/test_core_tasks.py:272]: assume EOPatches cannot be created without a bbox

File: core/eolearn/tests/test_core_tasks.py [hunk 2]
```diff
@@ -259,52 +260,25 @@ def test_initialize_feature(patch):
...
+    for feat in features:
+        assert feat in patch_dst
+        assert (id(patch[feat]) == id(patch_dst[feat])) != deep
 
-    assert (FeatureType.MASK_TIMELESS, "MTless1") in patch_dst
-    assert (FeatureType.MASK_TIMELESS, "MTless2") in patch_dst
+        if isinstance(patch[feat], np.ndarray):
+            assert_array_equal(patch[feat], patch_dst[feat])
+        else:
+            assert patch[feat] == patch_dst[feat]
```
Comment (zigaLuksic) [inline core/eolearn/tests/test_core_tasks.py:281]: i see that this does come up frequently in the module. 
Perhaps it should be extracted to a utility function (as you suggested).
The utility function can be part of this module for now. Or perhaps note it down and do a MR with these changes later.

File: core/eolearn/tests/test_core_tasks.py [hunk 3]
```diff
@@ -259,52 +260,25 @@ def test_initialize_feature(patch):
...
-
-    for i, feature in enumerate(features):
-        assert id(data[i]) != id(patch_dst[feature])
-        assert np.array_equal(data[i], patch_dst[feature])
-
-    features = [(FeatureType.MASK_TIMELESS, ...)]
-    patch_dst = EOPatch()
-    patch_dst = MoveFeatureTask(features)(patch_src, patch_dst)
+@pytest.mark.parametrize(
+    "features, deep",
```
Comment (zigaLuksic) [inline core/eolearn/tests/test_core_tasks.py:264]: why add a `deep` parameter here?
Do a second parametrization of the tests

Comment (jgersak) [reply core/eolearn/tests/test_core_tasks.py]: done

File: core/eolearn/tests/test_core_tasks.py (hunk 2 above)
Comment (jgersak) [reply core/eolearn/tests/test_core_tasks.py:308]: Issue opened i like to do it after current MR are merged.

File: core/eolearn/tests/test_core_tasks.py (hunk 1 above)
Comment (jgersak) [reply core/eolearn/tests/test_core_tasks.py]: done

**Pull request merged**